src/OperationResponse.php

```diff
@@ -456,12 +472,21 @@ public function getMetadata()
         return $metadata;
     }
 
-    private function operationsCall(string $method, string $requestClass)
+    /**
+     * Call the operations client to perform an operation.
+     *
+     * @param string $method The method to call on the operations client.
+     * @param string|null $requestClass The request class to use for the call.
+     *                                  Will be null for legacy operations clients.
+     */
+    private function operationsCall(string $method, ?string $requestClass)
     {
-        $firstArgument = $this->operationsClient instanceof OperationsClient
-            ? $requestClass::build($this->getName())
-            : $this->getName();
-        $args = array_merge([$firstArgument], $this->additionalArgs);
+        $args = array_merge([$this->getName()], array_values($this->additionalArgs));
+        if ($requestClass) {
+            $request = call_user_func_array($requestClass . '::build', $args);
```

Comment:
This assumes the request class has a builder method generated from the `google.api.method_signature`, right?

Reply:
Yes it does! And I've confirmed all our existing Request classes have this. However, if something were introduced which didn't support this (or someone supplied the wrong Request class), this would throw a PHP Fatal Error.

it may be a good idea to add an exception here when `build` doesn't exist on the request class, rather than potentially throwing a PHP fatal error.

Reply:
Done in 66e4a77e (for the other PR)